### DISCUSSION TITLE
easily add ticks to variable names in documentation?

### DISCUSSION THREAD

**Discussion by ElsLommelen** [inbo/checklist#130] [Q&A]
When adding the checklist functionality to an existing package, I notice a lot of 'spelling mistakes' in the documentation that are in fact variables that are not put between ticks.  Is there an easy way to solve these?  In case the same variable is used many times in the same file, I could use find and replace, but often each variable is only used once or twice in the function documentation.  Any suggestions to fix these without spending a lot of time with it?

**Comment by ThierryO** [comment]
The quick and (very) dirty fix would be to add them to the [custom dictionary](https://inbo.github.io/checklist/reference/custom_dictionary.html).
